doc/source/whatsnew/v0.24.0.txt

```diff
@@ -12,6 +12,8 @@ v0.24.0 (Month XX, 2018)
 
 New features
 ~~~~~~~~~~~~
+- The ``DataFrame`` method :func:`plot.scatter()` now accepts column names as an argument ``s`` to produce bubble plots in which the data in the corresponding column is represented by bubble sizes. (:issue:`16827`)
```

Comment:
The `:func:` won't work. Needs to be what's listed in our api.rst like 

    :func:`DataFrame.plot.scatter`

Reply:
"bubble plots" -> "scatter plots". Or maybe repharse the entire second half as "a plot where the marker sizes reflect the values in the column."

Reply:
OK - I change it to your formulation.